std_msgs/Header header
float32[] eye_center_3d
float32[] gaze_normal_3d
float32[] gaze_point_3d
float32[] norm_pos
float32 confidence
float32 timestamp

